lib/AST/ASTMutationFilter.cpp

```diff
@@ -0,0 +1,49 @@
+#include "mull/AST/ASTMutationFilter.h"
+
+#include "mull/MutationPoint.h"
+
+#include <llvm/IR/Instruction.h>
+#include <llvm/Support/FileSystem.h>
+
+using namespace mull;
+
+ASTMutationFilter::ASTMutationFilter(Diagnostics &diagnostics, ASTStorage &astStorage)
+    : diagnostics(diagnostics), astStorage(astStorage) {}
+
+bool ASTMutationFilter::shouldSkip(MutationPoint *point) {
+  llvm::Instruction *instruction = llvm::dyn_cast<llvm::Instruction>(point->getOriginalValue());
+  assert(instruction);
+
+  const mull::MutatorKind mutatorKind = point->getMutator()->mutatorKind();
+
+  const llvm::DebugLoc &debugInfo = instruction->getDebugLoc();
+  if (!debugInfo) {
+    return false;
```

Comment:
Shouldn't it be `true`?

Reply:
Good catch. It was a move from the old branch.